<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>KIPR's Instructional Software System</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="ui_centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="ui_tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QFrame" name="ui_projectFrame">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="ui_projectSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="kiss::project::ProjectView" name="ui_projects">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <property name="expandsOnDoubleClick">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="kiss::widget::InfoBox" name="ui_infoBox">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="lineWidth">
            <number>10</number>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>15</number>
           </attribute>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="OutputWidget" name="ui_errors" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="kiss::widget::CommunicationWidget" name="ui_comm" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="ui_statusbar"/>
  <widget class="QMenuBar" name="ui_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="ui_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OutputWidget</class>
   <extends>QWidget</extends>
   <header>output_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>kiss::widget::CommunicationWidget</class>
   <extends>QWidget</extends>
   <header>communication_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>kiss::project::ProjectView</class>
   <extends>QTreeView</extends>
   <header>project_view.hpp</header>
  </customwidget>
  <customwidget>
   <class>kiss::widget::InfoBox</class>
   <extends>QTableWidget</extends>
   <header>info_box.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/rc.qrc"/>
 </resources>
 <connections/>
</ui>
